### DISCUSSION TITLE
规则分组如果对同一网址有不同规则如何区分优先级？

### DISCUSSION THREAD

**Discussion by sd1114820** [vernesong/OpenClash#3903] [Q&A]
规则分组如果对同一网址有不同规则如何区分优先级？

**Comment by Nine499** [comment]
看配置文件，由上倒下

**Reply by sd1114820** [reply]
请问是上面的先执行，然后被下面的覆盖吗？还是上面的优先。

**Reply by rsgdn** [reply]
> 请问是上面的先执行，然后被下面的覆盖吗？还是上面的优先。

从上而下，匹配完就退出下一条，一直没有就漏网之鱼兜底
